sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -489,6 +489,38 @@ public void processElement(ProcessContext c) throws Exception {
   }
 
   @Test
+  public void testWriteEmptyPCollection() throws Exception {
```

Comment:
<!--thread_id:cc_140597879_t; commit:7202328ef0e10895b7c80bb5d1ec39244b09c6b8; resolved:0-->
<!--section:context-quote-->
> **jkff** wrote:
> Was there also a bug about writing empty collections that is addressed in this PR?

<!--section:body-->
I inherited this test from another PR. However we don't have any test coverage for writing an empty PCollection, so I left it in for the sake of coverage.